### ISSUE TITLE
Swagger UI breaks after changing sort method

### ISSUE THREAD

**Issue by blinchk** [SMILEY4/ktor-openapi-tools#16]
**Version:** 1.0.1

**Source code**
```kotlin
fun Application.configureSwagger() {
    install(SwaggerUI) {
        swagger {
            swaggerUrl = "swagger"
            sort = SwaggerUiSort.HTTP_METHOD
        }
    }
}
```

**Actual result**
Swagger returns JavaScript exception:
```
Uncaught ReferenceError: method is not defined
    at window.onload
```
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/49353043/212555755-645e0d94-35c0-4607-a31f-828a28a5fa73.png">

**Comment by blinchk** [comment]
Fixed in #16.

**Comment by blinchk** [comment]
Fixed in #18.
